glustercli/cmd/volume.go

```diff
@@ -247,6 +255,15 @@ var volumeGetCmd = &cobra.Command{
 		table.SetAlignment(tablewriter.ALIGN_LEFT)
 
 		for _, opt := range opts {
+			if flagGetBsc && opt.OptionLevel != "Basic" && !flagGetAdv {
```

Comment:
The conditions look a little complicated. What do you think of the following instead:

```go
if flagGetBsc && opt.OptionLevel == "Basic" {
    table.Append(...)
}
if flagGetAdv && opt.OptionLevel == "Advanced" {
    table.Append(...)
}
```

The above will handle case of a user giving `--basic`, `--advanced` and `--basic --advanced`.

Did you intend to show basic options also when `--advanced` is specified ?

Reply:
no, if the user provides only ```--advanced``` it will display only advance options. I have attached the CLI output also in this PR

Reply:
I have added this condition to avoid multiple ```table.Append(...)``` call inside if and else statements

Reply:
Okay. But the comment still stands though. You can achieve the same with the above example which has slightly better code readability.

Reply:
> I have added this condition to avoid table.Append(...) call inside if and else statements

I wouldn't mind that extra line. It makes code more clear to read.

Reply:
done